target/linux/ar71xx/base-files/lib/ar71xx.sh

```diff
@@ -287,11 +287,17 @@ tplink_board_detect() {
 	"360000"*)
 		model="TP-Link TL-WDR3600"
 		;;
-	"3C0001"*)
+	"3c0001"*)
 		model="OOLITE"
+
+		[ "$hwid" = "3c00010c" -a "$mid" = "00000001" ] && {
+			model="MINIBOX_V3.2"
+			hwver=""
```

Comment:
should you not set this to v3.2 ?

Reply:
I'm following the same rules of Minibox v1
https://github.com/p4u/source/blob/dc32cae3cc918719208502bad07f8075f6990e38/target/linux/ar71xx/base-files/lib/ar71xx.sh#L299